### DISCUSSION TITLE
how to disable italic font on tabline?

### DISCUSSION THREAD

**Discussion by SadSock** [LazyVim/LazyVim#2340] [Q&A]
i dislike italic font, thanks.
![image](https://github.com/LazyVim/LazyVim/assets/7643216/3e3a3814-c751-4b8b-981a-47e1539022e7)
sometimes it's ugly.

**Comment by dpetka2001** [comment]
```lua
-- ~/.config/nvim/lua/plugins/bufferline.lua
return {
  {
    "akinsho/bufferline.nvim",
    opts = {
      options = {
        style_preset = require("bufferline").style_preset.no_italic,
      },
    },
  },
}
```
Try this if it does what you want.

Edit: available options are at `:h bufferline-style-presets`.
